src/Cms/Language.php

```diff
@@ -224,6 +227,9 @@ public static function create(array $props)
 		// update the main languages collection in the app instance
 		App::instance()->languages(false)->append($language->code(), $language);
 
+		// trigger after hook
+		$kirby->trigger('language.create:after', compact('language', 'props'));
```

Comment:
Same as above. It should get the language and the input – not props.

```suggestion
		$kirby->trigger('language.create:after', ['language' => $language, 'input' => $props]));
```